### ISSUE TITLE
[BUG] incompatibility with meteor client

### ISSUE THREAD

**Issue by ghost** [CCBlueX/LiquidBounce#714]
### LiquidBounce Build/Version

nextgen

### Operating System

Windows 10

### Minecraft Version

1.18.1

### Describe the bug

Minecraft crash with Meteor client and Liquidbounce.

### Steps to reproduce

Launch minecraft with liquidbounce and meteor client.

### Client Log

```shell
[main/WARN]: @Redirect conflict. Skipping meteor-client.mixins.json:LivingEntityMixin->@Redirect::travelHasStatusEffectProxy(Lnet/minecraft/class_1309;Lnet/minecraft/class_1291;)Z from mod meteor-client with priority 1000, already redirected by liquidbounce.mixins.json:minecraft.entity.MixinLivingEntity->@Redirect::hookTravelStatusEffect(Lnet/minecraft/class_1309;Lnet/minecraft/class_1291;)Z from mod liquidbounce with priority 1000
```

**Comment by HowardZHY** [comment]
don't use 2 mixin hack clients together , their's mixins are conflict so this isn't likely to be solved

**Comment by ghost** [comment]
It most likely won't be solved. It's one of those clients that should not be used with LiquidBounce.

**Comment by FallenCrystal** [comment]
I took a look at all your issues. all r related to meteor client.
pls dont ask devs to do this. because LiquidBounce wasnt meant to do that. unless youre willing to submit your own code and pull the request. otherwise it will be permanently ignored

**Comment by 1zun4secondary** [comment]
bruh

**Comment by HowardZHY** [comment]
why did I send dumb issues XD



在 2023-02-20 00:09:33，"1zuna" ***@***.***> 写道：

Closed #714 as completed.
